docs/user_guide/index.md

```diff
@@ -48,6 +48,11 @@ version-dropdown
 search
 keyboard-shortcuts
 i18n
+back-to-top
+```
+
+```{toctree}
+
```

Comment:
Was the extra empty TOCtree necessary here? I'd expect it to be OK without:

```suggestion
```

Reply:
That is weird, I'm sorry - not sure where that came from.